apartment.rb

```diff
@@ -1,2 +1,24 @@
 class Apartment
+	attr_reader :rooms, :monthly_rent
+
+	def initialize(args={})
+		@rooms = args.fetch(:rooms," ")
+		@monthly_rent = args.fetch(:monthly_rent, 1000)
+	end
+
+	def total_sqft
+		rooms.map {|room| room.sqft}.reduce(:+)
```

Comment:
Another way to solve this is by just using the reduce function.
```
@rooms.reduce(0) { |sum, room| sum += room.sqft }
```
0 is the starting value and the block takes each rooms sqft and adds it to the sum value. The return value is the sum. @thomaslogangraves